Unable to list timereport when too many items

There is a limit to how many items we can use in a slack block message.
If we exceed that limit the request will fail.

This is what we will see in the logs:
```
[CRITICAL]	2020-06-18T13:33:04.569Z	9e8bdd6f-b527-4187-ac13-fa3eb8dac86f	Slack responded with not ok. Message was: {'ok': False, 'error': 'invalid_blocks', 'response_metadata': {'messages': ['[ERROR] no more than 50 items allowed [json-pointer:/blocks]']}}
```

--- a/chalicelib/lib/slack.py
+++ b/chalicelib/lib/slack.py
@@ -37,7 +37,10 @@ class Slack:
         if as_user is not None:
             data["as_user"] = as_user
 
-        data["blocks"] = self.blocks if self.blocks else None
+        data["blocks"] = self.blocks[:49] if self.blocks else None
+        if len(self.blocks) > 49:
+            data["text"] += " *Too much data to display, some data is missing"
+            data["text"] += ", select a shorter range.*"
         log.debug(f"Data is: ${data}")
 
         return self._handle_response(

